gpu/gpu_info_darwin.m

```diff
@@ -0,0 +1,11 @@
+//go:build darwin
+#include "gpu_info_darwin.h"
+
+uint64_t getRecommendedMaxVRAM()
+{
+	id<MTLDevice> device = MTLCreateSystemDefaultDevice();
+	NSLog(@"Metal default device %s", [device.name UTF8String]);
```

Comment:
Possible to remove the logging here as it's macOS-specific?

Reply:
Sure, the maximal VRAM is now logged via log/slog.